web/pgadmin/static/js/SchemaView/MappedControl.jsx

```diff
@@ -55,7 +55,7 @@ function MappedFormControlBase({ type, value, id, onChange, className, visible,
   case 'password':
     return <FormInputText name={name} value={value} onChange={onTextChange} className={className} type='password' inputRef={inputRef} {...props} />;
   case 'select':
-    return <FormInputSelect name={name} value={value} onChange={onTextChange} className={className} inputRef={inputRef} {...props} />;
+    return <FormInputSelect name={name} value={value == ''? []:value} onChange={onTextChange} className={className} inputRef={inputRef} {...props} />;
```

Comment:
why is this changed?

Reply:
The multi select dropdown was giving an empty option when cleared, So added that. But currently looking for a workaround on the publication code itself.

Reply:
Resolved